packages/fxa-payments-server/public/locales/en-US/main.ftl

```diff
@@ -51,6 +51,7 @@ settings-subscriptions-title = Subscriptions
 ## legal footer
 terms = Terms of Service
 privacy = Privacy Notice
+download-terms = Download Terms
```

Comment:
This doesn't match the localization id used in `TermsAndPrivacy`: "terms-download".